integration/tests/test_upgrade.py

```diff
@@ -29,6 +29,7 @@ def teardown_module(module):
     uninstall()
 
 
+@pytest.mark.skip(reason="Downgrade is failing. Please re-enable when fixed.")
```

Comment:
Should be able to remove this now, with the fixed version now released to universe?